recipes/scikit-optimize/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scikit-optimize" %}
+{% set version = "0.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "d8934c33e5958227d2ff97a4fae2134d2a00432b99e7f9d90b17e45cb1da9bb5" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - numpy
+    - scipy
+    - scikit-learn >=0.18
+    - matplotlib
+
+test:
+  imports:
+    - skopt
+    - skopt.learning
+    - skopt.learning.gaussian_process
+    - skopt.optimizer
+    - skopt.space
+
+about:
+  home: https://scikit-optimize.github.io/
+  license_file: {{ RECIPE_DIR }}/LICENSE.md
```

Comment:
Thanks for the extra data point. Those are important info to have. Still, there is no need to hold off  merging recipes that are OK and where the submitter may not have control. There is so much we can do :wink: 

BTW thanks for all the reviews! And feel free to ping me over those recipes that you find tricky. We are working on getting more people on board in the @conda-forge/staged-recipes team. (More forgiving merges is part of that effort :smile:)